td_core/copy_data/sitelinks.py

```
@@ -0,0 +1,225 @@
+            entities = json.loads(json.dumps(entities))
+            # ---
+            # print(entities)
+            # ---
+            all_entities.update(entities)
+    # ---
+    printe.output(f"wbgetentities result: {len(all_entities)}:")
+    # ---
+    return all_entities
+
+
+def get_qids_sitelinks(qs_list, qids_to_mdtitle={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)